src/endpoints/createLanguagesEndpoint.js

```diff
@@ -3,26 +3,37 @@
 import LanguageModel from '../models/LanguageModel'
 import EndpointBuilder from '../EndpointBuilder'
 import ParamMissingError from '../errors/ParamMissingError'
+import MappingError from '../errors/MappingError'
 import type { JsonLanguageType } from '../types'
 import Endpoint from '../Endpoint'
 
 export const LANGUAGES_ENDPOINT_NAME = 'languages'
 
 type ParamsType = { city: ?string }
 
-export default (baseUrl: string): Endpoint<ParamsType, Array<LanguageModel>> =>
+export default (baseUrl: string, isTunews: ?boolean): Endpoint<ParamsType, Array<LanguageModel>> =>
```

Comment:
A separate `createTunewsLanguageEndpoint` should be create here as the url differs.

This also simplifies the `withMapper` and does not touch the tests and the previous code.

Reply:
@maxammann Fixed